### DISCUSSION TITLE
The service of cdnjs cannot be used in mainland China

### DISCUSSION THREAD

**Discussion by f2ex** [cdnjs/cdnjs#14051] [General]
cdnjs is a great project, thank you very much for your services. However, in mainland China, it may be affected by certain factors, and it is currently impossible to use this service stably.
Although cloudflare has 17 networks in mainland China, cloudflare may be affected by G~F~W. This is really bad.
Will cdnjs take some measures for web services in mainland China?

**Comment by xtuc** [comment]
Thanks for your concerns! cdnjs should be available in China but we are not able to be served by china PoPs. What issues do you see?

**Reply by f2ex** [reply]
Sometimes such an error occurs, but it will recover after a few hours. I am not sure what the reason is. 
![error](https://i.imgur.com/8RNlrMX.png)

**Reply by f2ex** [reply]
> 
> 
> Thanks for your concerns! cdnjs should be available in China but we are not able to be served by china PoPs. What issues do you see?

New screenshot. Still unavailable, only use vpn to link.
![Image](https://i.imgur.com/6Qn67Bs.png)
